things/thing-7.md

```diff
@@ -0,0 +1,158 @@
+---
+HandheldFriendly: true
+author: ANDS
+description: |
+    Thing 7: Data citation for access & attribution. Citation analysis and
+    citation metrics are important to the academic community. Find out where
+    data fits in the citation picture.
+keywords: |
+    Thing 7, data citation, citation analysis, citation metrics,data,
+    research, research data, australia
+og:description: |
+    Thing 7: Data citation for access & attribution. Citation analysis and
+    citation metrics are important to the academic community. Find out where
+    data fits in the citation picture.
+og:site_name: ANDS
+og:title: 'Thing 7: Data citation for access &amp; attribution'
+og:type: website
+og:url: 'https://www.ands.org.au/working-with-data/skills/23-research-data-things/all23/thing-7'
+robots: 'index,follow'
+title: 'Thing 7: Data citation for access & attribution - ANDS'
+viewport: |
+    width=device-width, initial-scale=1.0, maximum-scale=1.0,
+    user-scalable=0
+---
+#Thing 7: Data citation for access & attribution
+
+**[&lt;&lt;Jump to Thing 6](thing-6.md) &nbsp; &nbsp; [Jump to Thing 8&gt;&gt;](thing-8.md)**
+
+Citation analysis and citation metrics are important to the academic
+community. Find out where data fits in the citation picture.
+
+-   **Getting started**: how to cite data
+-   **Learn more**: may the FORCE(11) be with you (while you cite data)
+-   **Challenge me**: are some data formats more likely to be cited?
+
+
+* [Getting started](#getting-started)
+* [Learn more](#learn-more)
+* [Challenge me](#challenge-me)
+
+## Getting started 
+
+### Citing research data
+
+Data citation continues the tradition of acknowledging other people’s
+work and ideas. Along with books, journals and other scholarly works, it
+is now possible to formally cite research datasets and even the software
+that was used to create or analyse the data.
+
+1.  Start by looking back to the [Weddell Seal
+    dataset](https://researchdata.ands.org.au/measuring-effects-human-leptonychotes-weddellii/640511 "Wendell seal dataset")
+    we explored in Thing 4.  Check out how many times it has been cited.
+    This citation count has been measured by Thomson Reuters Data
+    Citation Index product.
```

Comment:
Today this record displays no citation count.  Because the thing text contains "... citation count has been measured by Thomson Reuters... ", I would expect to see a count.  In comparison this Weddell Seal dataset record does contain a citation count.  Suggest replacing the original URL with one that points to a record which illustrates a citation count.

Reply:
Good catch, have updated the URL.